Makefile

```diff
@@ -109,44 +112,21 @@ init-gohan:
 	\
 	if [ ! -d "./gohan" ]; then \
 		echo "-- Cloning Gohan --" ; \
-		git clone ${GOHAN_REPO} ; \
+		git clone ${GOHAN_REPO} --depth 1 -b ${TAG_OR_BRANCH} ; \
```

Comment:
the `--depth 1` parameter is probably something that will hurt in the future due to how Gohan and bento_public are working.

In production container, this allows to set up a directory that contains only the latest version of the code without the full git history. It's an optimization that avoids downloading information that will never be used. In dev mode, the developer needs to create the git repo first and the full git history is the best way for doing it because... well... you need to see what has happened with the code, right?!
In gohan and bento_public, there is a middle ground between dev and prod which makes it so that the local repo is used for both purposes. Hence we need the full git history.